BoardGameTrainer/Program.cs

```diff
@@ -194,11 +196,12 @@ private static void CreateNewGame()
             if (gameNum == 0)
             {
                 gameManager = new CheckersManagerFactory()
-            .CreateGameManager(new UctFactory(1.41), new IterationStopCondition(10000));
+                    .CreateGameManager(new UctFactory(1.41), new IterationStopCondition(1000));
```

Comment:
powinno być TimeStopCondition zależne od wczytanej z formularza wartości

Reply:
Nie mamy jeszcze napisanego TimeStopCondition, więc pókki co jest to co jest